src/bornhack/urls.py

```diff
@@ -15,6 +16,8 @@
 from tickets.views import ShopTicketListView
 from bar.views import MenuView
 
+admin.site.login = login_required(admin.site.login)
```

Comment:
What's the purpose here? The admin already has login required :)

Reply:
it's your plugin you tell me! 😛 In the documentation it says that we need to wrap the admin login like this, otherwise an admin user can just login at /admin without the 2fa token!

Reply:
Ah great! That's something that's been added after I worked on it :smile:

Reply:
Maybe throw a comment in there saying why we decorate it?